.docker/bin/dsh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+
+help ()
+{
+	echo "dsh - Docker Shell"
```

Comment:
Let's call this Drude Shell, since it is not docker-only, but for the whole infrastructure.